documentation/source/user_section/tutorials/registration-to-template/lumbar-registration/labeling.rst

```diff
@@ -0,0 +1,74 @@
+   :align: center
+
+Above is a comparison of sample results, demonstrating that registration will vary significantly depending on which landmarks are chosen. The "ideal result" will depend on the goals of your research. However, in most cases, you will want to stick with one approach and use it consistently throughout your research.
+
+Creating an image with 1 disc label and 1 conus medullaris label
+================================================================
+
+As a demonstration, this section will show you how to manually label one disc (T9-T10) as well as the conus medullaris.
+
+There are multiple recommended ways of manually creating labels:
+
+1. Using ``sct_label_utils -create-viewer``, which will provide you with a GUI window to select point-wise labels.
+2. Determining the coordinates by inspecting the image using a separate 3D image viewer, then adding them via the command line using ``sct_label_utils -create``
+3. Using the "Edit Mode -> Create Mask" options of a 3D image viewer such as FSLeyes.
+
+Option 1 is the easiest to use when voxel-perfect accuracy is not 100% necessary. However, if you need to precisely locate a specific voxel, we recommend options 2 or 3.
+
+Adding the labels via Option 2: Image Viewer + ``sct_label_utils -create``
+--------------------------------------------------------------------------
+
+In this case, voxel accuracy **is** important, because the conus medullaris label must overlap with the spinal cord segmentation. (This is because of the "straightening step" during registration -- the straightening transform is limited to the voxels covered by the spinal cord segmentation, so the registration labels must also exist in that region in order for the labels to be straightened.)
+
+Because of the necessary precision, we recommend that you use Option 2 by following these steps:
+
+- Open the anatomical image in an image viewer such as FSLeyes, then overlay the spinal cord segmentation on top.
+- Move your cursor to the tapered end of the spinal cord.
+- Adjust the coordinate in all 3 axes to ensure the the cursor is within 0-1 voxel of the edges of the segmentation.
+- Take note of the coordinate, and provide it to the command below.
+
+For this image, the coordinate ``[27,79,80]`` seems to be appropriate location:
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lumbar-registration/label-selection-conus-medullaris.png
+   :align: center
+
+Repeat the same process for the posterior tip of the T9-T10 disc. Here, we will use the coordinate ``[22,77,187]``:
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lumbar-registration/label-selection-T9-T10-disc.png
+   :align: center
+
+The command below will open the input image in a GUI viewer, allowing you to select points to add labels to. In this case, we will add labels for the T9-T10 disc (numerical ID 17) and the conus medullaris (numerical ID 60).
```

Comment:
There is no GUI since it direclty creates the labels at the coordinates we specified